packages/framework/src/Form/Admin/Product/ProductEditFormType.php

```diff
@@ -50,16 +50,23 @@ class ProductEditFormType extends AbstractType
      */
     private $pluginDataFormExtensionFacade;
 
+    /**
```

Comment:
If I were a nitpick, I would say that this not belong into this commit... but I won't :smile:

Reply:
thanks :nail_care: